internal/view/helm_history.go

```diff
@@ -56,6 +59,9 @@ func (h *History) bindKeys(aa ui.KeyActions) {
 
 	aa.Delete(ui.KeyShiftA, ui.KeyShiftN, tcell.KeyCtrlS, tcell.KeyCtrlSpace, ui.KeySpace, tcell.KeyCtrlD)
 	aa.Add(ui.KeyActions{
+		ui.KeyY:      ui.NewKeyAction(yamlAction, h.yamlCmd, true),
+		ui.KeyD:      ui.NewKeyAction("Describe", h.describeCmd, true),
+		ui.KeyV:      ui.NewKeyAction("Values", h.valuesCmd, true),
```

Comment:
I think we could leverage a `valueExtender` (see views/xxx_extender) here and inject this action for both chart/history. Is both dao implement the Valuer interface I think the might be an option. 
Would this make sense?